cypress-tests/cypress/jsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
+      "Texts/*": ["./constants/texts/*"],
+      "Selectors/*": ["./constants/selectors/*"]
+    }
+  }
+}
```

Comment:
You shoud include cypress modules for auto-completion.

Similary set allowJs to true

Refer the jsconfig part in https://docs.cypress.io/guides/tooling/IDE-integration#Set-up-in-your-Dev-Environment

Reply:
Updated.